docs/api/assets.md

```diff
@@ -0,0 +1,422 @@
+
+These `AssetCSS` or `AssetJS` objects are then available on the `.css` or `.js`
+properties of the [`Podlet Response`](layout.md#podlet-response) object returned
+by the `.client.fetch()` method or emitted by the `beforeStream` event when the
+`.client.stream()` method is used.
+
+Example of a layout fetching a podlet and printing the assets of the podlet:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Express-->
+
+```js
+const podlet = layout.client.register({
+    name: 'myPodlet',
+    uri: 'http://localhost:7100/manifest.json',
+});
+
+app.get(layout.pathname(), async (req, res, next) => {
+    const incoming = res.locals.podium;
+
+    const response = await podlet.fetch(incoming);
+
+    console.log(response.css)  // array with the podlets AssetCSS objects
+    console.log(response.js)   // array with the podlets AssetJS objects
+
+    [ ... ]
+});
+```
+
+<!--Hapi-->
+
+```js
+const podlet = layout.client.register({
+    name: 'myPodlet',
+    uri: 'http://localhost:7100/manifest.json',
+});
+
+app.route({
+    method: 'GET',
+    path: layout.pathname(),
+    handler: (request, h) => {
+        const incoming = request.app.podium;
+
+        const response = await podlet.fetch(incoming);
+
+        console.log(response.css)  // array with the podlets AssetCSS objects
+        console.log(response.js)   // array with the podlets AssetJS objects
+
+        [ ... ]
+    },
+});
+
+app.start();
+```
+
+<!--Fastify-->
+
+```js
+const podlet = layout.client.register({
+    name: 'myPodlet',
+    uri: 'http://localhost:7100/manifest.json',
+});
+
+app.get(layout.pathname(), async (request, reply) => {
+    const incoming = reply.app.podium;
+
+    const response = await podlet.fetch(incoming);
+
+    console.log(response.css)  // array with the podlets AssetCSS objects
+    console.log(response.js)   // array with the podlets AssetJS objects
+
+    [ ... ]
+});
+```
+
+<!--HTTP-->
+
+```js
+const podlet = layout.client.register({
+    name: 'myPodlet',
+    uri: 'http://localhost:7100/manifest.json',
+});
+
+const server = http.createServer(async (req, res) => {
+    let incoming = new HttpIncoming(req, res);
+    incoming = await layout.process(incoming);
+
+    const response = await podlet.fetch(incoming);
+
+    console.log(response.css)  // array with the podlets AssetCSS objects
+    console.log(response.js)   // array with the podlets AssetJS objects
+
+    [ ... ]
+});
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+When a layout fetches one or more podlets its common that one would like to
+include the assets to these podlets in the full HTML document the layout is
+composing.
+
+By setting the [`Podlet Response`](layout.md#podlet-response) objects from the
+podlets one fetch on to the `HttpIncoming.podlets` property in a layout one
+appends the `AssetCSS` and `AssetJS` objects for the requested podlets onto the
+`HttpIncoming.css` and `HttpIncoming.js` properties.
+
+This way; the layouts registered assets and the assets for all the requested
```

Comment:
```suggestion
This way the layouts registered assets and the assets for all the requested
```
The `;` should be used when two sentences can be both stand-alone, and separated by a comma.